trigger/testTimer/timer.go

```diff
@@ -0,0 +1,245 @@
+package testTimer
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/project-flogo/core/data/metadata"
+	"github.com/project-flogo/core/support/log"
+	"github.com/project-flogo/core/trigger"
+)
+
+var triggerMd = trigger.NewMetadata(&HandlerSettings{}, &Output{})
+
+const (
+	Resume = iota + 1
+	Pause
+	Start
+	Stop
+)
+
+func init() {
+	trigger.Register(&Trigger{}, &Factory{})
+}
+
+type Factory struct {
+}
+
+// Metadata implements trigger.Factory.Metadata
+func (*Factory) Metadata() *trigger.Metadata {
+	return triggerMd
+}
+
+// New implements trigger.Factory.New
+func (*Factory) New(config *trigger.Config) (trigger.Trigger, error) {
+	s := &Settings{}
+
+	err := metadata.MapToStruct(config.Settings, s, true)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Trigger{settings: s}, nil
+}
+
+type Trigger struct {
+	handlers []*Handler
+	logger   log.Logger
+	settings *Settings
+	router   *httprouter.Router
+	ch       chan int
+}
+type Handler struct {
+	handler  trigger.Handler
+	settings *HandlerSettings
+}
+
+// Init implements trigger.Init
+func (t *Trigger) Initialize(ctx trigger.InitContext) error {
+
+	t.ch = make(chan int)
+
+	t.logger = ctx.Logger()
+
+	if t.settings.Control {
+		router := httprouter.New()
+
+		//Register For All.
+		router.POST("/control/resume/", resumeHandler(t.ch, nil))
+		router.POST("/control/pause/", pauseHandler(t.ch, nil))
+		router.POST("/control/start/", startHandler(t.ch, nil))
+		router.POST("/control/stop/", stopHandler(t.ch, nil))
+		for _, handler := range ctx.GetHandlers() {
```

Comment:
Maybe change "control" to "tester"
		
router.POST("/tester/resume", resumeHandler(t.ch, nil))
router.POST("/tester/pause", pauseHandler(t.ch, nil))
router.POST("/tester/start", startHandler(t.ch, nil))
router.POST("/tester/stop", stopHandler(t.ch, nil))

Maybe use the following pattern and have the handler switch on id.  You can get the value of the 'id' from httprouter.Params 

router.POST("/tester/resume/:id", resumeHandler(t.ch, nil))
router.POST("/tester/pause/:id", pauseHandler(t.ch, nil))
router.POST("/tester/start/:id", startHandler(t.ch, nil))
router.POST("/tester/stop/":id, stopHandler(t.ch, nil))